release-notes/opensearch-ml-common.release-notes-2.13.0.0.md

```diff
@@ -0,0 +1,54 @@
+## Version 2.13.0.0 Release Notes
+
+Compatible with OpenSearch 2.13.0
+
+
+### Experimental Features
+* hidden agent ([#2204](https://github.com/opensearch-project/ml-commons/pull/2204))
+* Initiate MLInferencelngestProcessor ([#2205](https://github.com/opensearch-project/ml-commons/pull/2205))
+* auto deployment for remote models ([#2206](https://github.com/opensearch-project/ml-commons/pull/2206))
+* support question answering model ([#2208](https://github.com/opensearch-project/ml-commons/pull/2208))
+* Guardrails for remote model input and output ([#2209](https://github.com/opensearch-project/ml-commons/pull/2209))
+
+### Enhancements
+* Add verification to rate limiter number field ([#2113](https://github.com/opensearch-project/ml-commons/pull/2113))
+* Support Pretrained Model from on-prem or different cloud provider ([#2119](https://github.com/opensearch-project/ml-commons/pull/2119))
+* asymmetric embeddings ([#2123](https://github.com/opensearch-project/ml-commons/pull/2123))
+* Set the number of ml system index primary shards to 1 ([#2137](https://github.com/opensearch-project/ml-commons/pull/2137))
+* prevent exposing internal ip when an agent gets an internal OpenSearch exception ([#2154](https://github.com/opensearch-project/ml-commons/pull/2154))
+* Suppress removal AccessController in java.security has been deprecated and marked for removal ([#2195](https://github.com/opensearch-project/ml-commons/pull/2195))
+* add remote predict thread pool ([#2207](https://github.com/opensearch-project/ml-commons/pull/2207))
+
+### Infrastructure
+
+
+### Bug Fixes
+* fix error code when executing agent ([#2120](https://github.com/opensearch-project/ml-commons/pull/2120))
+* fix npe when executing agent with empty parameter ([#2145](https://github.com/opensearch-project/ml-commons/pull/2145))
+* fix cohere chat blueprint ([#2167](https://github.com/opensearch-project/ml-commons/pull/2167))
+* fix delete model cache on macOS causing model deploy fail with model ([#2180](https://github.com/opensearch-project/ml-commons/pull/2180))
+* adding BWC for connector config field ([#2184](https://github.com/opensearch-project/ml-commons/pull/2184))
+* Bump common-compress package to fix CVE ([#2186](https://github.com/opensearch-project/ml-commons/pull/2186))
```

Comment:
CVE is not bug fix. , move to `Maintenance` ?

Reply:
done